core/src/main/java/hudson/util/ColorPalette.java

```diff
@@ -47,7 +47,7 @@
     /**
      * Color list usable for generating line charts.
      */
-    @SuppressFBWarnings("MS_SHOULD_BE_FINAL")
+    @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", justification = "used in several plugins")
```

Comment:
Technically not "used in several plugins" because no plugins are _currently_ using this field, but I think it makes sense to keep it non-final because it seems useful to expose to plugins. So I would update the justification to read: "exposed to plugins".